package.json

```
@@ -26,7 +26,7 @@
     "node": "20.14.0"
   },
   "scripts": {
-    "dev": "env-cmd -f .env.development node server.js",
+    "dev": "next dev",
```

```suggestion
    "dev": "env-cmd -f .env.development next dev",
```
이렇게 하셔야, env가 제대로 적용될 것 같습니다.

---

감사합니다~